### PR TITLE
Background adjust

### PR DESCRIPTION
**Pull request by DominikDR**

### CODE REVIEW

File: background/setBackground.js [hunk 1]
```diff
@@ -1,5 +1,38 @@
-const setBackground = (weatherCode) => {
-	const background = document.getElementById("background");
-	background.style.backgroundImage = `url(${backgroundsMapping[weatherCode]})`;
-	background.style.backgroundColor = '#efefef';
+const background = document.getElementById("background");
+
+const toggleBackground = (weatherCode) => {
+    if (weatherCode) {
+        setWeatherBackground(weatherCode);
+    }  else {
```
Comment (Dammic) [inline background/setBackground.js]: Please use consistent indentation

File: background/setBackground.js [hunk 2]
```diff
@@ -1,5 +1,38 @@
...
+        setInitialBackground();
+    }
+}
+
+const setWeatherBackground = (weatherCode) => {
+    const initialBackground = document.getElementById('initial-background');
+    if (document.body.contains(initialBackground)) {
```
Comment (Dammic) [inline background/setBackground.js]: Why are you checking twice if element is in DOM. document.getElementById will be undefined if it's not in DOM. So instead, you can do:
```
if (initialBackground)
```

File: background/setBackground.js [hunk 3]
```diff
@@ -1,5 +1,38 @@
...
+        document.body.removeChild(initialBackground);
+    }
+    background.style.display = 'flex';
+    background.style.backgroundImage = `url(${backgroundsMapping[weatherCode]})`;
```
Comment (Dammic) [inline background/setBackground.js]: Instead of changing styles, operate on classes

File: background/setBackground.js [hunk 4]
```diff
@@ -1,5 +1,38 @@
...
+        type:"image/svg+xml",
+        data: "./assets/Initial_background/worldMap.svg",
+        alt: "SVG world map.",
+    });
+    
+    const divContainer = document.getElementById("initial-background");
+    divContainer.innerHTML = `
+    <div id="headline">
+        <span>Check the weather around the World.</span>
+    </div>
```
Comment (Dammic) [inline background/setBackground.js]: indentation

File: background/setBackground.js [hunk 5]
```diff
@@ -1,5 +1,38 @@
...
+    `;
+    divContainer.appendChild(finalObject);
+    
+    background.style.display = 'none';
```
Comment (Dammic) [inline background/setBackground.js]: please modify classes instead of adding styles

File: cssApp.css [hunk 6]
```diff
@@ -50,14 +53,57 @@ html, body {
...
+    max-height: 100%;
+    filter: blur(1.5px);
+}
+#headline {
+    animation: enlargeText 0.7s ease-out;
+    position: absolute;
+    top: 100px;
+    width: 50%;
+    margin: 0 auto;
+    max-width: fit-content;
```
Comment (Dammic) [inline cssApp.css]: Be aware that this will not work on all browsers.

File: cssApp.css [hunk 7]
```diff
@@ -50,14 +53,57 @@ html, body {
...
+}
+#world-map-background {
+    position: absolute;
+    max-height: 100%;
+    filter: blur(1.5px);
+}
+#headline {
+    animation: enlargeText 0.7s ease-out;
+    position: absolute;
+    top: 100px;
```
Comment (Dammic) [inline cssApp.css]: move top property to where you declare left/right

File: background/setBackground.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (DominikDR) [reply background/setBackground.js]: What is wrong in this part?

File: background/setBackground.js (hunk 2 above)
Comment (DominikDR) [reply background/setBackground.js]: Right :+1:

File: background/setBackground.js (hunk 4 above)
Comment (DominikDR) [reply background/setBackground.js]: :+1:

File: cssApp.css (hunk 7 above)
Comment (DominikDR) [reply cssApp.css]: ok

File: background/setBackground.js (hunk 5 above)
Comment (DominikDR) [reply background/setBackground.js]: In this function I'll use:
`background.classList.add('hide-content');`
and where I want to show the content:
`background.classList.remove('hide-content');`
Will it be ok?

File: cssApp.css (hunk 6 above)
Comment (DominikDR) [reply cssApp.css]: Do you mean top property?
`https://caniuse.com/#search=top`

**Pull request merged**